azurelinuxagent/common/osutil/factory.py

```diff
@@ -38,6 +38,13 @@
 
 from distutils.version import LooseVersion as Version
 
+UBUNTU_20_04_IMAGE_PATH = "/etc/os-release/ub20-image"
+
+def get_python_symlink_path_if_ubuntu_20_04_image():
```

Comment:
should be in osutils, rather than in the factory

Reply:
Moved to osutil/ubuntu.py, looked like the perfect place for it to be